modules/oxide/src/less/theme/components/view/view.less

```diff
@@ -30,22 +30,6 @@
     position: relative;
   }
 
-  .tox-view__label {
-    color: @text-color;
-    font-weight: @button-font-weight;
-    line-height: @button-line-height;
-    padding: @button-padding-y @button-padding-x;
-    text-align: @button-text-align;
-  }
-
-  .tox-view__label--normal {
-    font-size: @font-size-md;
-  }
-
-  .tox-view__label--large {
-    font-size: @font-size-lg;
-  }
```

Comment:
Wonder if these should be kept here, not sure if we can use the variables within the plugin itself?

Reply:
Hmmm, I am not sure how we have handled this in the past. I don't think we can use the variables but I think it is acceptable to have a little bit of duplication and hardcode the font-size values in the plugin if needed

Reply:
👍, will move the normal/large label classes into the plugin itself.